spec/requests/api/v0/rg_post_article_spec.rb

```diff
@@ -0,0 +1,45 @@
+RSpec.describe Api::V0::ArticlesController, type: :request do
+
+  describe 'research group POST article' do 
+    let(:rg_user) { FactoryBot.create(:user, role: :research_group) }
```

Comment:
Please avoid using abbreviations in variable names. Why not just call it `research_group`?

Reply:
All the naming of the variables and assertions have been corrected.